dev-lang/rust/metadata.xml

```diff
@@ -1,17 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="project">
-    <email>rust@gentoo.org</email>
-    <name>Rust Project</name>
-  </maintainer>
-  <use>
-    <flag name="cargo">Install cargo component</flag>
-    <flag name="clippy">Install clippy component</flag>
-    <flag name="system-llvm">Use the system LLVM install</flag>
-    <flag name="rls">Install rls component</flag>
-    <flag name="rustfmt">Install rustfmt component</flag>
-    <flag name="wasm">Build support for the wasm32-unknown-unknown
-    target</flag>
-  </use>
+	<maintainer type="project">
+		<email>rust@gentoo.org</email>
+		<name>Rust Project</name>
+	</maintainer>
```

Comment:
The metadata should reflect smauel as the maintainer, the rust project for gentoo has no clue as to his build system